snare/tests/test_parse_tanner_response.py

```diff
@@ -54,6 +56,36 @@ def test_parse_type_one(self):
         expected_result = [self.page_content, self.content_type, {}, 200]
         self.assertCountEqual(real_result, expected_result)
 
+    def test_parse_type_one_query(self):
+        self.requested_name = '/?'
+        self.detection = {"type": 1}
+        self.call_content = b'<html><body></body></html>'
+
+        async def test():
+            (self.res1, self.res2,
+             self.res3, self.res4) = await self.handler.parse_tanner_response(self.requested_name, self.detection)
+
+        self.loop.run_until_complete(test())
+        real_result = [self.res1, self.res2, self.res3, self.res4]
+        expected_result = [self.page_content, self.content_type, {}, 200]
+        self.assertCountEqual(real_result, expected_result)
+
+    def test_parse_type_one_error(self):
+        self.detection = {"type": 1}
+        meta_content = {"/index.html": {}}
```

Comment:
This can be removed.

Reply:
Yes, even though I have simulated the test-case using `something/` but there is no test that demonstrates this case which is also a possibility. 
But your decision would be final, should I remove this?

Reply:
This case need not be handled here, it can be checked in `cloner.py`. So I think it's good to keep it simple here.

Reply:
Okay,